packages/calculator-bn.js/etc/calculator-bigint.api.md

```diff
@@ -0,0 +1,59 @@
+## API Report File for "@dinero.js/calculator-bn.js"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { BinaryOperation } from '@dinero.js/core';
+import { ComparisonOperator } from '@dinero.js/core';
+import { UnaryOperation } from '@dinero.js/core';
+
+// @public
+export const add: BinaryOperation<BN>;
+
+// @public (undocumented)
+export const calculator: {
+    add: BinaryOperation<BN, BN>;
+    compare: BinaryOperation<BN,
+    ComparisonOperator
+    >;
+    decrement: UnaryOperation<BN, BN>;
+    increment: UnaryOperation<BN, BN>;
+    integerDivide: BinaryOperation<BN, BN>;
+    modulo: BinaryOperation<BN, BN>;
+    multiply: BinaryOperation<BN, BN>;
+    power: BinaryOperation<BN, BN>;
+    subtract: BinaryOperation<BN, BN>;
+    zero: typeof zero;
+};
+
+// @public
+export const compare: BinaryOperation<BN, ComparisonOperator>;
+
+// @public
+export const decrement: UnaryOperation<BN>;
+
+// @public
+export const increment: UnaryOperation<BN>;
+
+// @public
+export const integerDivide: BinaryOperation<BN>;
+
+// @public
+export const modulo: BinaryOperation<BN>;
+
+// @public
+export const multiply: BinaryOperation<BN>;
+
+// @public
+export const power: BinaryOperation<BN>;
+
+// @public
+export const subtract: BinaryOperation<BN>;
+
+// @public
+export function zero(): BN;
+
+// (No @packageDocumentation comment for this package)
+
+```
```

Comment:
I don't know what this file is doing here, it has the wrong name. The `api.md` files are supposed to be built by `api-extractor`. 

- Clear the `etc` directory
- `rm lib` - clean the ts declaration directory
- `yarn build:types`
- commit the file it creates.

Reply:
removed 1c3e6dc86c3c437b05a972abd44ff1724f251a32